hash/apple2gs_flop_orig.xml

```diff
@@ -4086,4 +4079,150 @@ license:CC0-1.0
 		</part>
 	</software>
 
+	<software name="sinbadfal">
+		<description>Sinbad and the Throne of the Falcon</description>
+		<year>1990</year>
+		<publisher>Cinemaware</publisher>
+		<info name="usage" value="Requires a 768K Apple IIgs ROM1 or later."/>
+		<sharedfeat name="compatibility" value="A2GS" />
+		<!-- Dump released: 2022-11-20 -->
+		<!--"Sinbad and the Throne of the Falcon" is a 1990 action game developed by Curt Toumanian (art director), Steve Quinn (original art), Jeff Hilbers (art), Rob Landeros (art), Russ Truelove (art), Bill Williams (music & story), Jim Simmons (orchestration), Andrew Caldwell (orchestration), Douglass Sharp (computagraphy), and distributed by Cinemaware.-->
+		<part name="flop1" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 1 Reel 1" />
+			<dataarea name="flop" size="1296414">
+				<rom name="sinbad and the throne of the falcon - reel 1.woz" size="1296414" crc="a3056453" sha1="f90a3f48445f80abe28ba368ff9de51deb55930a" />
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 2 Reel 2 " />
+			<dataarea name="flop" size="1296381">
+				<rom name="sinbad and the throne of the falcon - reel 2.woz" size="1296381" crc="3eb2b298" sha1="1920ab0321f8bcf85a607baf2b3a0778251c3747" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="magmyths">
+		<description>Magical Myths</description>
+		<year>1988</year>
+		<publisher>Unicorn Software</publisher>
+		<info name="usage" value="Requires a 768K Apple IIgs."/>
+		<sharedfeat name="compatibility" value="A2GS" />
+		<!-- Dump released: 2022-11-24 -->
+		<!--"Magical Myths" is a 1988 educational program developed by Stan Brewster, Joseph B. Hewitt IV (graphics), and June Stark, and distributed by Unicorn Software.-->
+		<part name="flop1" interface="floppy_3_5">
+			<feature name="part_id" value="Disk  Side " />
+			<dataarea name="flop" size="1296701">
+				<rom name="magical myths iigs disk 1.woz" size="1296701" crc="681789c6" sha1="6f65fcfe7f25eeac365e17fe00fb9c044048dc04" />
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<feature name="part_id" value="Disk  Side " />
+			<dataarea name="flop" size="1296189">
+				<rom name="magical myths iigs disk 2.woz" size="1296189" crc="aff8db40" sha1="e9fb62fb6006938b101cde428bd9252d333944e4" />
+			</dataarea>
+		</part>
```

Comment:
(Magical Myths line 4112) The two disks have identical part IDs – since this is used as the display name, it will confuse users and make switching disks difficult.